airbyte-integrations/connectors/destination-snowflake/build.gradle

```diff
@@ -44,4 +46,6 @@ integrationTestJava {
 dependencies {
     implementation 'net.snowflake:snowflake-jdbc:3.14.1'
     implementation 'org.apache.commons:commons-text:1.10.0'
+    implementation 'org.json:json:20210307'
```

Comment:
We don't need this.  We already have Jackson dependency in our dependency chain.